### DISCUSSION TITLE
change save icon colour if the project is currently unsaved

### DISCUSSION THREAD

**Discussion by emhl** [xournalpp/xournalpp#3575] [Ideas]
afaik when using xournal++ in fullscreen mode there is no way to know if the project currently has been saved or not (apart from exiting fullscreen mode or hitting save :D) . 
changing the save icons colour would be an easy way to show that information.

**Comment by LittleHuba** [comment]
Great suggestion. @Fatih20 what do you think as our UI-Design-Guru? 😁 

Should we just change the color or would you rather like a different icon there?
